app/models/comment.rb

```diff
@@ -0,0 +1,11 @@
+class Comment < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] Comment should belong to author not user i.e `belongs_to :author, class_name: 'User'`